Studs/Events/ViewModels/EventsViewModel.swift

```diff
@@ -35,14 +35,19 @@ final class EventsViewModel {
 
     func fetchData() {
         // Fetch the events
-        Http.fetchAllEvents().subscribe(onNext: { events in
+        Http.fetchAllEvents().subscribe(onNext: { [weak self] events in
             // Sort events, earliest first
```

Comment:
You can do `guard let self = self else { return }` here and then you don't need to do `self?`. You might have to write it as
> guard let \`self\` = self else { return }